File: Java/1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] sol = new int[2];
        for (int i = 0; i < nums.length; i++)
         for (int n = i + 1; n < nums.length; n++){
            if (nums[n] + nums[i] == target){
                sol[0] = n;
                sol[1] = i;
                return sol;
            }
         }
         return sol;
    }
}

File: Python/1-two-sum/two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        sol = []
        count_n = -1
        for n in nums:
            count_i = -1
            count_n += 1
            for i in nums:
                count_i += 1
                if not count_n == count_i:
                    if (n + i) == target:
                        sol.append(count_n)
                        sol.append(count_i)
                        return sol
        

        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        